gradio/interface.py

```diff
@@ -362,7 +362,7 @@ def __init__(
         self.local_url = None
 
         self.favicon_path = None
-        utils.version_check()
+        analytics.version_check()
```

Comment:
This check won't work now if analytics is disabled. @abidlabs

Reply:
Yes I think that's an acceptable sacrifice to ensure no network requests to gradio servers